ush/ufsda/genYAML.py

```diff
@@ -72,7 +72,7 @@ def get_runtime_config(config_dict):
     bkg_tstep = str(config_dict.get('BKG_TSTEP', f"PT{assim_freq}H"))
     component_dict = {
         'atmos': 'ATM',
-        'chem': 'AERO',
```

Comment:
this should stay chem because that is the COMROT structure for NCO.

Reply:
I'm getting a key error "aero" otherwise. Changing this to "aero" fixed it. So I need another solution.

Reply:
in `test/genYAML_prep_aero.sh` what if you change it to `chem`? Does it still error? The directory structure will be like `gdas.20230303/12/chem` so we need it to be chem.